pandas/formats/format.py

```diff
@@ -1756,13 +1758,501 @@ def __init__(self, row, col, val, style=None, mergestart=None,
                               "vertical": "top"}}
 
 
+class CSSWarning(UserWarning):
+    """This CSS syntax cannot currently be parsed"""
```

Comment:
i think either out the css in css.py and excel stuff in excel.py (u can rip out the ExcelFormatter class from formats); prob just need to change a couple of imports to make it work